beta/nncf/tensorflow/sparsity/rb/operation.py

```diff
@@ -0,0 +1,97 @@
+"""
+ Copyright (c) 2021 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import tensorflow as tf
+
+from beta.nncf.tensorflow.functions import logit
+from beta.nncf.tensorflow.layers.custom_objects import NNCF_CUSTOM_OBJECTS
+from beta.nncf.tensorflow.layers.operation import InputType
+from beta.nncf.tensorflow.layers.operation import NNCFOperation
+from beta.nncf.tensorflow.sparsity.magnitude.functions import apply_mask
+from beta.nncf.tensorflow.sparsity.rb.functions import calc_rb_binary_mask, st_binary_mask, binary_mask
+
+OP_NAME = 'rb_sparsity_mask_apply'
+
+@NNCF_CUSTOM_OBJECTS.register()
+class RBSparsifyingWeight(NNCFOperation):
+
+    def __init__(self, eps=1e-6):
+        '''
+        :param eps: minimum value and the gap from the maximum value in
+            distributed mask
+        '''
+        super().__init__(name=OP_NAME,
+                         trainable=True)
+        self.eps = eps
+
+    # TODO: make it static
```

Comment:
Please, delete this comment. I would like to suggest leaving as-is.

Reply:
Done